nixos/lib/test-driver/test-driver.py

```diff
@@ -978,6 +977,37 @@ def subtest(name: str) -> Iterator[None]:
     return False
 
 
+def _test_symbols() -> Dict[str, Any]:
+    general_symbols = dict(
+        os=os,
+        log=globals().get("log"),  # extracting those symbol keys
+        machines=globals().get("machines"),  # without being initialized
+        start_all=start_all,
+        create_machine=create_machine,
+        # run_tests=run_tests,
+        # join_all=join_all,
+        test_script=globals().get("test_script"),  # same
+        subtest=subtest,
+        serial_stdout_off=serial_stdout_off,
+        serial_stdout_on=serial_stdout_on,
+    )
+    return general_symbols
+
+
+def test_symbols() -> Dict[str, Any]:
+
+    general_symbols = _test_symbols()
+
+    machine_symbols = {m.name: machines[idx] for idx, m in enumerate(machines)}
+    print(
+        "Available Symbols:\n    "
+        + ", ".join(map(lambda m: m.name, machines))
```

Comment:
I don't find the message really clear for the test `simple.nix`:
```
Available Symbols:
    machine
    -------
    os, log, machines, start_all, create_machine, test_script, subtest, serial_stdout_off, serial_stdout_on
```
It's not obvious what `machine` is (moreover it looks like a title:/).

What about something such as:
```
additional exposed symbols:
    machine,
    start_all, test_script, machines, log, os, create_machine, subtest, serial_stdout_on, serial_stdout_off
```

I sorted items in the `general_symbols` dict definition (dict are sorted since python3.7) to start with most used symbols (totally arbitrary).